include/experimental/__p0009_bits/maybe_static_value.hpp

```diff
@@ -61,18 +61,18 @@ namespace experimental {
 namespace detail {
 
 // static case
-template <class _T, _T __v,
-          _T __is_dynamic_sentinal = dynamic_extent,
+template <class _dynamic_t, class static_t, _static_t __v,
+          _static_t __is_dynamic_sentinal = dynamic_extent,
```

Comment:
Just a spelling error:
```suggestion
          _static_t __is_dynamic_sentinel = dynamic_extent,
```

Reply:
I will search for all of those later.